interview-questions.md

```diff
@@ -9,59 +9,58 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: 'this' references a state object in React. 
 
-  Researched answer:
+  Researched answer: 'this' is a keyword in React that refers to an instance of a component class. 
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a library comprised of JavaScript components, or blocks of code, that form the features of a webpage. 
 
-  Researched answer:
+  Researched answer: React is a JavaScript library made up of reusable blocks of code known as components, which are used to build webpages.
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: The lifestyle method that is required in a React class component is the constructor method. 
 
-  Researched answer:
+  Researched answer: The constructor method is a required lifestyle method in a React class component. 
```

Comment:
Since React is always looking to show or _render_ something on a page, the lifecycle method that React class components require is the render() method.